### DISCUSSION TITLE
Can't trigger a manual Sync anymore

### DISCUSSION THREAD

**Discussion by SirGamsay** [abraunegg/onedrive#2720] [Q&A] [answered]
So, today I updated to a newer version of the OneDrive Client to `onedrive v2.4.25-1+np2+4.1`.
In older Versions, when I needed a file fast on my other machines I just used `onedrive --synchronize` to trigger an upload/ download manually on the devices and I was finished.

Now after the Update, when I try this It just says `ERROR: onedrive application is already running - check system process list for active application instances`

After reading a few Discussions I first thought this was because of a custom configuration, but even with the default config this happens.

I understand, that this may have been bad practice, but a feature like that came in Handy often times.

Is there a similar option to trigger a manual sync in the newer versions?

**Comment by abraunegg** [comment, marked as answer]
@SirGamsay 
> `ERROR: onedrive application is already running - check system process list for active application instances`
> ...
> Is there a similar option to trigger a manual sync in the newer versions?

You do not say what your older version was ..

All older versions contained a bug - as if you were able to run the client whilst other instances against the *same* configuration was active, you would have been destroying the database and any updates that the *running* instance was performing.

Essentially you have a background process running the client - it will be a systemd service most likely. As you are running Ubuntu or a clone of Ubuntu given the client version, you *may* have the Ubuntu default systemd service running which would be causing this issue.

The other alternative is that you enabled the systemd services yourself rather than the Ubuntu default one.

In both cases, you need to stop the systemd service running the client in the background, then do your manual sync.

There is no option here other than to stop your systemd services first.

**Reply by SirGamsay** [reply]
Thank you very much, for the quick help.
I couldn't provide my older version, because I did the Update on both devices already, so I can't look back and see what version I have been using.
Back then sometimes the manual triggering of the application provided something like `cant sync right now, database is locked by another process` I kind of thought, this took care of the Database destroying thing, but this seemingly wasn't enough.

Yes I do have a onedrive service running in the background, which works well too.
Then I will just stop the service in the background, in the few occasions I need it to.

Thank you again for the quick help.
